### DISCUSSION TITLE
Migrating to Hugo modules

### DISCUSSION THREAD

**Discussion by diemol** [google/docsy#1419] [Q&A] [answered]
👋 everyone,

I've been trying to migrate https://www.selenium.dev/ to Hugo modules. The basic migration steps work, but everything that lives under the `assets` directory is not loading, which means that our side looks like the default Docsy site.

After reading [this comment](https://github.com/google/docsy/discussions/1095#discussioncomment-3583542), I saw that one needs to mount some directories to have things working properly, but after doing that, I got an error while doing `hugo serve`.

```
Start building sites … 
hugo v0.101.0-466fa43c16709b4483689930a4f9ac8add5c9f66+extended darwin/arm64 BuildDate=2022-06-16T07:09:16Z VendorInfo=gohugoio
Error: Error building site: TOCSS: failed to transform "scss/main.scss" (text/x-scss): "/var/folders/85/m0jdxw2j3yn9mrn_2dpnbsmm0000gn/T/hugo_cache/modules/filecache/modules/pkg/mod/github.com/google/docsy@v0.6.0/assets/scss/_blog.scss:5:13": The target selector was not found.
Use "@extend .-bg-orange !optional" to avoid this error.
Built in 4753 ms
```

I currently have this work in progress as a PR: https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1308

Any help and/or pointers on what to check would be appreciated. Thank you!

**Comment by deining** [comment, marked as answer]
I had a look at your project and realized you came pretty far with the conversion already! The problems you are enountering are due to the fact that your project specific file `/assets/scss/_colors.scss` overwrites the colors defined in the docsy theme. Since only a subset of all colors needed is defined in your file, the build fails. By changing the file name of your custom color file to `_colors_project.sccs` and importing this file, you can overcome the difficulties you are encountering.

**Related PR:** https://github.com/SeleniumHQ/seleniumhq.github.io/pull/1310

**Reply by diemol** [reply]
Thank you, @deining! Really appreciate it 🎉
